docs/docs-content/clusters/public-cloud/aws/eks-hybrid-nodes/create-hybrid-node-pools.md

```diff
@@ -0,0 +1,369 @@
+   click on **Presets** on the right-hand side, and select **Agent Mode**.
+
+8. Click **Next layer** to continue.
+
+9. Select **Nodeadm** as your base Kubernetes pack, and click **Next**.
+
+10. On the **Configure Pack** page, under **Pack Version**, select your Kubernetes version from the **drop-down Menu**.
+
+11. In the YAML editor, make any changes you need for the kubelet or containerd configuration. Refer to
+    [Amazon EKS Hybrid Nodes Configuration](https://github.com/aws/eks-hybrid?tab=readme-ov-file#configuration) for
+    guidance on the available options.
+
+12. Click **Next layer** to continue.
+
+13. Select **Custom CNI** as your base Network pack, and click **Next**.
+
+14. In the YAML editor on the **Configure Pack** page, change the value of `manifests.byo-cni.contents.data.custom-cni`
+    from `calico` to `dummy`.
+
+    :::info
+
+    While this change is not required for the pack to function, setting it to `dummy` better indicates that this pack
+    serves as a placeholder only. This is because the Container Network Interface (CNI) was already created for hybrid
+    nodes during the [Add CNI Cluster Profile](./import-eks-cluster-enable-hybrid-mode.md#add-cni-cluster-profile)
+    steps.
+
+    :::
+
+15. Click **Confirm** when complete.
+
+16. In **Profile Layers**, click **Next** to continue.
+
+17. Click **Finish Configuration**.
+
+Your cluster profile for hybrid nodes is now created and can be used in the
+[Create Hybrid Node Pool](#create-hybrid-node-pool) steps.
+
+### Validate
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu**, select **Profiles**.
+
+3. On the **Profiles** page, click on your newly created profile.
+
+4. Check that your profile layers are present and correct.
+
+## Create Hybrid Node Pool
+
+### Prerequisites
+
+- An Amazon EKS cluster imported with hybrid mode enabled. Refer to
+  [Import EKS Cluster and Enable Hybrid Mode](./import-eks-cluster-enable-hybrid-mode.md) for guidance.
+
+- Edge hosts have been registered with Palette through
+  [Agent Mode](../../../../deployment-modes/agent-mode/agent-mode.md) or by using
+  [provider images](../../../edge/edgeforge-workflow/palette-canvos/build-provider-images.md).
+
+  :::warning
+
+  If using provider images, you must include the following in your `.arg` file during the
+  [build steps](../../../edge/edgeforge-workflow/palette-canvos/build-provider-images.md#build-provider-images).
+
+  ```shell
+  K8S_DISTRIBUTION=nodeadm
+  K8S_VERSION=<kubernetesVersion>  # supported versions: [ 1.28.0 | 1.29.0 | 1.30.0 | 1.31.0 ]
+  ```
+
+  Replace `<kubernetesVersion>` with your version of Kubernetes. For example, `1.29.0`.
+
+  :::
+
+- A cluster profile created for your hybrid nodes. Refer to
+  [Create Cluster Profile for Hybrid Node Pools](#create-cluster-profile-for-hybrid-node-pools) for steps.
+
```

Comment:
What do you think about adding a high level verification step to these prereqs?

> Network connectivity verification:
> - Verify that both tunnels are up for the site-to-site VPN connection (if using a VPN)
> - Verify that a debug pod scheduled on one of the EKS cluster's cloud worker nodes can ping the edge host(s)
> - Verify that the edge host(s) can ping the private IP of an EC2 instance in the EKS cluster's VPC

Reply:
Sure, will add these in @TylerGillson.